### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by gitnyasha**
**What's inside**
- a newsweek website clone created using bootstrap

### CODE REVIEW

**Comment by izaiasneto4** [comment]
### Hi @gitnyasha, good work so far. There's some minor things for us to change before getting your project approved :arrow_down: 

- [ ] Some of your images look stretched and out of proportion in the mobile version of your website
image

- [ ] You're missing two link names from the bottom :wink: 
image

- [ ] Your logo at the top is not at the proportion of the original one in the desktop version and is also in a different color, let's change that and I'll make your project look even better:+1. Have you considered using [SVG](https://www.w3schools.com/graphics/svg_intro.asp) to solve these issues :wink:?
Yours
image

Original page
image

#### Please, next time **do not** close the pr as you did with #2. Let's make the changes and submit in the same pr unless you're asked to do it differently :wink: 

Please, when finished with the changes re-submit this PR for a new review :muscle: 

TSE: Izaias Neto

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+
+- Bootstrap
+
+## Authors
+
+- https://github.com/gitnyasha
+
+## Live Demo
+
+[Live Demo](https://gitnyasha.github.io/using-bootstrap/)
```
Comment (Oluwadamilareolusakin) [inline README.md]: Remember to update this with an active live link. Use the tool in my comments if you don't have a resource you prefer or check it out regardless, I think you'll love it

**Comment by Oluwadamilareolusakin** [comment]
The major issues I perceived are with your navigation bar, and I have attached some photos below to illustrate the issues:
At about 1024px
screenshot

At about 768px
screenshot

At about 375px
screenshot
screenshot

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

#### Missing footer links
As @izaiasneto4 rightly pointed out, you have some links missing in your footer, please refer to his comments about this and make appropriate changes

#### Live link
Please remember to update the live link in your project to point to the actual website. My go to resource for making live links is [Raw Githack](https://raw.githack.com) Please check it out, it's free and easy to use :+1: 

### Summary
Your project is beautiful in desktop mode, however, the design breaks down as we move to smaller screen sizes and as such, the project can't be approved. Please look into these issues thoroughly and request another review! Until next time :smiley:, Best of luck!

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

File: README.md (hunk 1 above)
Comment (gitnyasha) [reply README.md]: give a clear a review please the images are too small

**Comment by Oluwadamilareolusakin** [comment]
Please don't close PRs

**Comment by Oluwadamilareolusakin** [comment]
Trying to fix the images,  they're unusually small

File: README.md (hunk 1 above)
Comment (gitnyasha) [reply README.md]: how is adding more links to the footer relevant to the project

**Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.**

Comment (Oluwadamilareolusakin) [reply README.md]: Haha, surely you understand that these projects are not just for "Microverse". Please understand that future employers WILL go through your repositories and you never know what they may open up and as such, it's important to put your best foot forward on every single project. We are just trying to help you do that.

Comment (Oluwadamilareolusakin) [reply README.md]: All pictures have been updated to larger versions.

**Pull request closed without merging**